src/component/handlers/edit/editOnInput.js

```diff
@@ -88,8 +89,41 @@ function editOnInput(editor: DraftEditor): void {
   var domSelection = global.getSelection();
 
   var {anchorNode} = domSelection;
-  if (anchorNode.nodeType !== Node.TEXT_NODE) {
-    return;
+  const isNotTextNode =
```

Comment:
P.S. this entire naming scheme is incredibly confusing, because `draft_killswitch_allow_nontextnodes` is actually saying to *not* allow *non* text nodes...